apps/arena/priv/config.json

```diff
@@ -49,21 +49,116 @@
           "x": -6044.0,
           "y": -1439.0
         },
-        "radius": 1567.0
+        "radius": 1567.0,
+        "shape": "circle"
       },
       {
         "position": {
           "x": -860.0,
           "y": 5338.0
         },
-        "radius": 1090.0
+        "radius": 1090.0,
+        "shape": "circle"
       },
       {
         "position": {
           "x": -1630.0,
           "y": 4588.0
         },
-        "radius": 317.0
+        "radius": 317.0,
+        "shape": "circle"
+      },
+      {
```

Comment:
Is there a PR on the front? I think we should remove (or maybe comment) the obstacles until the client has a representation for them.

Reply:
whoops, yes this was something that we added to test on a playtest some days ago. i deleted this obstacles